projects/web-wallet/rpc-protocol.md

```diff
@@ -0,0 +1,330 @@
+# 1. Introduction
```

Comment:
Add a link to the Wallet Connect documentation, please. I'm wondering where we validate the network, and how dApps might test their integration using the public testnet or even a private network. Maybe add some words about these topics (or make a reference to the Wallet Connect documentation).

Reply:
Added the link to the wallet-connect RFC and to the Web3Wallet docs in 8b170640a819a1888546d2b0960201ef695bf01a

I also added a new RPC, to get the wallet's current connected network in c53a287158be9f89d99f8c60bffec59cddc57693

What do you think?

Reply:
Great! Thanks!